lib/pbench/server/filetree.py

```diff
@@ -320,25 +320,59 @@ def get_metadata(self) -> JSONOBJECT:
             self.metadata = {s: dict(metadata.items(s)) for s in metadata.sections()}
         return self.metadata
 
-    def subprocess_run(self, command: str, unpacked: Path):
+    @staticmethod
+    def subprocess_run(command: str, curr_dir: Path, exception: Exception, path: Path):
         """
         Runs command under subprocess.run
 
         Args:
-            command: command to be executed
-            unpacked: unpacked Directory
+            command: command to be executed.
+            curr_dir: Current Directory where tarball needs to be unpacked.
+            exceptio: exception that needs to be raised.
```

Comment:
@dbutenhof, I think you're reviewing old revision (from 7 days ago), here.

Reply:
Yes, I started out going one commit at a time to see what had changed, and dropping comments as I went. In several cases my comments were partially obsoleted or substantively changed by subsequent commits, but not enough to completely invalidate them. In this case, Riya fixed the parameter name typo, but I still don't like the description which references the specific expected exception types by name when that's not actually a requirement of the interface; along with I think still failing to clarifying that the parameter is a class rather than an instance.